src/components/HorizontalThermometer/RiskLevelThermometer/RiskLevelThermometer.style.tsx

```diff
@@ -25,15 +25,15 @@ export const ColorBlock = styled.div<{ color: string }>`
 
   &:first-of-type {
     border-radius: 99px 0 0 99px;
-    margin-left: 0.75rem;
+    margin-left: 0.5rem;
```

Comment:
Had to tweak margins and the thermometer bar width to get it to fit on the county map.  It's a bit cramped, but works. 🤷 

![image](https://user-images.githubusercontent.com/206364/151465909-375df366-27ae-4b2d-9c73-82c0adc051cb.png)